CITests/run

```diff
@@ -14,6 +15,8 @@ cd /package
 if [ "$ONLY_RUN_SWIFT_LINT" == 'yes' ]; then
     ls
     swiftlint
+elif [ "$DOCKER_IMAGE_TAG" == 'swift:5.4.0-amazonlinux2' ]; then
+   swift build -Xswiftc -Xfrontend -Xswiftc -sil-verify-none -c release
```

Comment:
nit: Can we add a comment here explaining what this does and why it is required so we can refer back to it later?